static-pages/latest.md

```diff
@@ -1,22 +1,22 @@
 title: Latest News
 
-![Cover of yellow envelope with drawn portrait of Lincoln in the top left corner and a depiction of Lincoln chopping wood in the middle](/static/img/LincolnCampaign.jpg)
 
 ## Letters to Lincoln Challenge 
 ---
 
-### Help us transcribe 30,000+ pages of the Abraham Lincoln Papers by the end of 2018!
-
+![Cover of yellow envelope with drawn portrait of Lincoln in the top left corner and a depiction of Lincoln chopping wood in the middle](/static/img/LincolnCampaign.jpg)
 
-We’re welcoming you to crowd.loc.gov with a grand challenge. You might guess that Abraham Lincoln Papers include materials written in his own hand, but did you know the collection contains correspondence sent to Abraham Lincoln throughout his life and political career? 
+### Help us transcribe 10,000+ items from the Abraham Lincoln Papers by the end of 2018!
 
-Here's a taste of what  A range of materials by writers ranging from friends and associates from Lincoln’s Springfield days, well-known political figures and reformers, constituents writing to their President, and even the occasional document in Lincoln's own hand - read the concerns and requests of 19th century Americans and international correspondents. 
+#### A grand challenge: why we're asking you to join us
+Around half of the digitized Abraham Lincoln Papers, primarily materials written by Lincoln, have been transcribed by other volunteers at Knox College and elsewhere, and are already keyword searchable at loc.gov. However, there remain 10,000+ items including letters and other materials sent to him that are not yet keyword searchable. Completing the Letters to Lincoln Challenge will make all of the digital Lincoln Papers word-searchable and accessible to future readers. Just imagine the possiblities--from new research to local connections--that will be possible once we've achieved this goal. Thank you in advance for sharing your time with us. Your Community Managers, reference librarians and curatorial staff here at the Library of Congress will be cheering you on with bonus historical context and resources all along the way, as well as some special rewards for goals met!
```

Comment:
I would generally favor using a true em dash (—) instead of `--`, and adding spaces around both for typographic reasons and to avoid browsers word-wrapping the adjacent words as one long chunk.